src/EnergyPlus/Autosizing/WaterHeatingCoilUASizing.cc

```diff
@@ -141,10 +140,7 @@ Real64 WaterHeatingCoilUASizer::size(EnergyPlusData &state, Real64 _originalValu
                         this->addErrorMessage(msg);
                         ShowContinueError(state, msg);
                     }
-                    this->dataErrorsFound = true;
                 } else if (SolFla == -2) {
-                    this->errorType = AutoSizingResultType::ErrorType1;
-                    errorsFound = true;
```

Comment:
Failed SolveRoot with a -1 for iterations exceeded or -2 for bad limits are now not an error but instead a diagnostic, with messages and sizing data to aid the user in a resolution,